modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -69,6 +72,9 @@ public int getValue() {
         VERSION,
         CONFIG,
         INFO,
+        QUERY_BLOCK,
```

Comment:
This ordering does not reflect the intended priority. For consistency, the location should match the ordering in both `call`, `getBreakingTaskPriority` and `getSkippedTasks`.

Reply:
revised.